### ISSUE TITLE
Remove unused modules: archetypes, minimal-starter ?

### ISSUE THREAD

**Issue by scottkurz** [eclipse-ee4j/starter#191]
Are the archetypes, minimal-starter modules of any use now, since they're not configured in the top-level reactor?   Are we just waiting for someone to remove them ?   Or am I missing the usage?  Thanks

**Comment by m-reza-rahman** [comment]
In the 2.0 release, we have added a deprecation notice to all these. Once 2.0 is released into Maven Central, the idea is to remove these from the repository.

**Comment by ojuschugh1** [comment]
should I remove the unused module -archetypes,minimal-starter from pom.xml in the root directory? Will it be a  significant change for PR  sir?

**Comment by m-reza-rahman** [comment]
No. The committers will handle this once the 2.0 release is stabilized. This is why this is assigned to me and does not have a 'help wanted' label.

**Comment by ojuschugh1** [comment]
ohh thanks for letting me know !!

**Comment by m-reza-rahman** [comment]
Fixed by: https://github.com/eclipse-ee4j/starter/pull/260
